lib/builders/windows-builder.ts

```diff
@@ -121,27 +121,34 @@ const defaultOptions: WindowsOptions = {
 
 const defaultKarpenterProps: addons.KarpenterAddOnProps = {
```

Comment:
let's run `make lint` to remove this warning that defaultKarpenterProps is assigned a value but never used. 
Also is it an indication of a defect? why default props are never used?

Reply:
This was leftovers from previous commit from another PR. Can remove as I don't see the necessity.

Reply:
addressed